util-images/sleep/Makefile

```diff
@@ -0,0 +1,17 @@
+PROJECT = k8s-testimages
```

Comment:
Please change to `PROJECT = k8s-staging-perf-tests`

Reply:
Done